rma/report/rma_report_templates.xml

```diff
@@ -84,19 +84,23 @@
                 <table class="table table-condensed">
                     <thead>
                         <tr>
-                            <th>Product</th>
-                            <th class="text-right">Quantity</th>
-                            <th class="text-right">Unit Price</th>
+                            <th t-name="product" invisible="1">Product</th>
```

Comment:
t-name is a directive in qweb reserved for putting a name to a template, this is not a correct use. To improve extensibility an easy xpath, you can simple use `name`

Reply:
Oh, I didn't know it, updated! Thx Lois!